cfgov/tccp/jinja2/tccp/includes/fields.html

```diff
@@ -17,3 +17,15 @@
         else (default if default is not none else "None")
     }}
 {%- endmacro -%}
+
+{%- macro lower_first_letter(string) -%}
+    {{ string[0].lower() ~ string[1:] }}
+{%- endmacro -%}
```

Comment:
Handle empty string cases or where the string is only one letter long:

```suggestion
{%- macro lower_first_letter(s) -%}
    {{- (s[0].lower() ~ (s[1:] if s | length > 1)) if s -}}
{%- endmacro -%}
```